py_swf/clients/workflow.py

```diff
@@ -65,3 +66,182 @@ def terminate_workflow(self, workflow_id, reason):
             workflowId=workflow_id,
             reason=reason,
         )
+
+    def count_open_workflow_executions(
+            self,
+            oldest_start_date,
+            latest_start_date=None,
+            workflow_name=None,
+            version=None,
+            tag=None,
+            workflow_id=None,
+    ):
+        """
+        Count the number of open workflows for a domain. You can pass in filtering criteria.
+        The results are best effort and may not exactly reflect recent updates and changes.
+        http://docs.aws.amazon.com/amazonswf/latest/apireference/API_CountOpenWorkflowExecutions.html
```

Comment:
protip: our sphinx configuration is set up so that you can link to the external SWF docs. Example:
`Passthrough to :meth:`~SWF.Client.count_open_workflow_executions``